wgpu-core/src/device/mod.rs

```diff
@@ -2738,6 +2738,15 @@ impl<A: HalApi> Device<A> {
             if let Some(e) = error {
                 return Err(pipeline::CreateRenderPipelineError::DepthStencilState(e));
             }
+
+            if ds.bias.clamp != 0.0
+                && !self
+                    .shared
+                    .downlevel_flags
+                    .contains(wgt::DownlevelFlags::DEPTH_BIAS_CLAMP)
```

Comment:
This could be self.require_downlevel_flags(wgt::DownlevelFlags::DEPTH_BIAS_CLAMP) I believe

Reply:
Right, copy-pasted this from the vulkan file - not sure why rust analyzer didn't complain.